cluster/helm/splice-validator/templates/validator.yaml

```diff
@@ -108,8 +108,28 @@ spec:
         {{ else -}}
         - name: ADDITIONAL_CONFIG_BFT_SCAN
           value: |
+            {{- if .Values.scanClientConfig }}
+              {{- if eq .Values.scanClientConfig.scanType "trust-single" }}
+            canton.validator-apps.validator_backend.scan-client.type = "trust-single"
+            canton.validator-apps.validator_backend.scan-client.url = {{ index .Values.scanClientConfig.seedUrls 0 | quote }}
+              {{- else}}
+                {{- if and (hasKey .Values.scanClientConfig "useLastKnownConnectionsForInitialization") (not .Values.scanClientConfig.useLastKnownConnectionsForInitialization)}}
```

Comment:
Sorry to come back to this but thinking about it, it doesn't actually make sense to make this a first class citizen of the helm charts. We shouldn't need to ever disable this unless somehow it's broken and then we can just set the ADDITIONAL_CONFIG env variables. So i would fully remove this one from helm